misk/src/main/kotlin/misk/web/jetty/WebActionsServlet.kt

```diff
@@ -75,6 +80,9 @@ internal class WebActionsServlet @Inject constructor(
       }
     } catch (_: ProtocolException) {
       // Probably an unexpected HTTP method. Send a 404 below.
+    } catch (e: Throwable) {
+      log.error { "Uncaught exception on ${request.dispatchMechanism()} ${request.httpUrl()}" }
+      throw e
```

Comment:
should this just gracefully return a 500 below? otherwise you'll get duplicate log messages right? one ERROR, one WARN

Reply:
I'm not totally sure of semantics if this crashes after something already started writing a response. I guess we can't fail any worse if we are already here so 👍